src/tools/wix/ChainPackageInfo.cs

```diff
@@ -611,7 +611,23 @@ private void ResolveMsiPackage(BinderFileManager fileManager, Dictionary<string,
 
                     if (!Common.IsValidModuleOrBundleVersion(this.Version))
                     {
-                        this.core.OnMessage(WixErrors.InvalidProductVersion(this.PackagePayload.SourceLineNumbers, this.Version, sourcePath));
+                        // not a proper .NET version (i.e., five fields); can we get a valid three-part version number?
```

Comment:
I think we have the technology for that...but the bug says three. Why does the bug say three? We should fire the scribe. I can't think of a reason not to take all four parts except that major upgrades aren't going to work either way...